tests/unit/src/main/scala/tests/BaseWorksheetLspSuite.scala

```diff
@@ -489,6 +491,33 @@ abstract class BaseWorksheetLspSuite(scalaVersion: String)
     } yield ()
   }
 
+  test("definition".only, withoutVirtualDocs = true) {
```

Comment:
This is my bad, `.only` is one of the default tags that will run only the test case tagged with it and it breaks on CI in case someone (such as me here) forgets to remove it :sweat: